### ISSUE TITLE
Keyboard navigation does not work at all in Device Info (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9187]
## Observed behavior
Keyboard navigation doesn't work in the Device Info page. Once a user is in `DeviceInfo`, no focus comes into place.

## Errors and logs
None

## Expected behavior
Focus ring should appear first on the server URL link, and then be looped back to the app bar.

## User-facing consequences
Keyboard users will not be able to navigate.

## Steps to reproduce
1. Sign in as admin
2. Use mouse to navigate to Device
3. Use mouse to navigate to Info
4. Use keyboard to try to navigate the page

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
Closing as Mac settings issue
